conf/references/mlss_conf.xml

```diff
@@ -57,94 +62,194 @@
-      <genome name="oryctolagus_cuniculus"/>
+      <genome name="Oryctolagus_cuniculus"/>
       <!-- Rodents -->
       <genome name="mus_musculus"/>
       <genome name="rattus_norvegicus"/>
       <genome name="sciurus_vulgaris"/>
-      <!-- Primates -->
-      <genome name="macaca_mulatta"/>
-      <genome name="pan_troglodytes"/>
-      <genome name="callithrix_jacchus"/>
-      <genome name="prolemur_simus"/>
-      <!-- Carnivora -->
-      <genome name="canis_lupus_familiaris"/>
-      <genome name="felis_catus"/>
-      <genome name="suricata_suricatta"/>
-      <!-- Afrotheria -->
-      <genome name="echinops_telfairi"/>
+      <genome name="chinchilla_lanigera"/>
+      <genome name="dipodomys_ordii"/>
+      <genome name="jaculus_jaculus"/>
+      <genome name="peromyscus_maniculatus_bairdii"/>
+      <genome name="Nannospalax_galili"/>
+    </collection>
+
+    <collection name="actinopterygii">
+      <base_collection name="shared_vertebrates"/>
+      <!-- Outgroups -->
+      <genome name="ciona_intestinalis"/>
+      <genome name="eptatretus_burgeri"/>
+      <genome name="amblyraja_radiata"/>
+      <!-- Polypteriformes  -->
+      <genome name="erpetoichthys_calabaricus"/>
+      <!-- Lepisosteiformes -->
+      <genome name="lepisosteus_oculatus"/>
+      <!-- 	Osteoglossiformes -->
+      <genome name="paramormyrops_kingsleyae"/>
+      <genome name="scleropages_formosus"/>
+      <!-- Clupeiformes -->
+      <genome name="denticeps_clupeoides"/>
+      <genome name="clupea_harengus"/>
+      <!-- Siluriformes -->
+      <genome name="ictalurus_punctatus"/>
+      <!-- Gymnotiformes -->
+      <genome name="electrophorus_electricus"/>
+      <!-- Characiformes -->
+      <genome name="astyanax_mexicanus"/>
+      <genome name="pygocentrus_nattereri"/>
+      <!-- Esociformes -->
+      <genome name="esox_lucius"/>
+      <!-- Salmoniformes -->
+      <genome name="salmo_trutta"/>
+      <genome name="oncorhynchus_mykiss"/>
+      <!-- Perciformes -->
+      <genome name="dicentrarchus_labrax"/>
+      <genome name="sparus_aurata"/>
+      <!-- Cypriniformes -->
+      <genome name="cyprinus_carpio"/>
+      <!-- Gadiformes -->
+      <genome name="gadus_morhua"/>
+      <!-- Beryciformes -->
+      <genome neme="myripristis_murdjan"/>
+      <!-- 	Tetraodontiformes -->
+      <genome name="takifugu_rubripes"/>
+      <!-- Anabantiformes -->
+      <genome name="betta_splendens"/>
+      <!-- Pleuronectiformes -->
+      <genome name="scophthalmus_maximus"/>
+      <!-- Scorpaeniformes -->
+      <genome name="cyclopterus_lumpus"/>
+      <!-- Cichliformes -->
+      <genome name="astatotilapia_calliptera"/>
+      <!-- 	Elopiformes -->
+      <genome name="megalops_cyprinoides"/>
+      <!-- Beloniformes -->
+      <genome name="oryzias_latipes"/>
+    </collection>
+
+    <collection name="sauropsida">
+      <base_collection name="shared_vertebrates"/>
+      <!-- Outgroup -->
+      <genome name="ciona_intestinalis"/>
```

Comment:
```suggestion
```
As before

Reply:
cannot be removed because not in shared vertebrates